h2o-core/src/main/java/hex/DistributionFactory.java

```diff
@@ -389,7 +389,7 @@ public double deviance(double w, double y, double f) {
         if (Math.abs(y - f) <= _huberDelta) {
             return w * (y - f) * (y - f); // same as wMSE
         } else {
-            return 2 * w * (Math.abs(y - f) - _huberDelta) * _huberDelta; // note quite the same as wMAE
+            return (2 * w * Math.abs(y - f) - _huberDelta) * _huberDelta; // (2 * wMAE - delta) * delta
```

Comment:
I wonder if this is correct. Let's assume that `w=1.5` and check if the function is really continuous at the `huberDelta` value (`(y-f) == huberDelta`).

I get `1.5*huberDelta**2` on one side and `(3*huberDelta - huberDelta)*huberDelta == 2*huberDelta**2` on the other one.

```suggestion
            return  w * (2 * Math.abs(y - f) - _huberDelta) * _huberDelta; // (2 * wMAE - delta) * delta
```